README.md

```diff
@@ -90,14 +85,9 @@ and health-checking mechanisms.
 
 ### Build and install `hare` rpm from source.
 **NOTE: If you have built Motr and HARE from source you will not need to generate RPM packages below**
-* Install Motr.
-
-  * .. from RPMS
-    ```sh
-    sudo yum -y install cortx-motr cortx-motr-devel
-    ```
+* Build `Motr` RPMs.
 
-  * .. or
+  * from soruce
```

Comment:
typo

Reply:
made changes as part of #1610 pull request.